### PR TITLE
Add debugging guide and issue template for bug reports

### LINKED ISSUE #585: Update TROUBLESHOOTING with information on how to debug
To make things easier for us and for users we should update the troubleshooting guide (and maybe add a issue template) to include information on how to debug if things go wrong.

Here are some great resources: #584

### PR DESCRIPTION
**Pull request by lgeiger**
Fixes #585

### CODE REVIEW

**Comment by rgbkrk** [comment]
Mega 👍

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+[Description of the issue]
+
+**Steps to Reproduce:**
+
+1. [First Step]
+2. [Second Step]
+
+**Versions:**
+
+Which OS and which version of Atom and Hydrogen are you running?
```
Comment (n-riesco) [inline .github/ISSUE_TEMPLATE.md]: Perhaps, suggest running `apm --version`

Comment (n-riesco) [inline .github/ISSUE_TEMPLATE.md]: and `apm list`

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
+# Troubleshooting
+
+## Debugging Guide
+
+Sometimes unexpected things happen.
+In this case please open a [Issue](https://github.com/nteract/hydrogen/issues) and be sure to include the Atom version and the output or a screenshot of the debug logs.
```
Comment (n-riesco) [inline TROUBLESHOOTING.md]: an issue

Comment (n-riesco) [inline TROUBLESHOOTING.md]: suggest `apm --version` and `apm list`

File: TROUBLESHOOTING.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+
+Therefor Hydrogen provides helpful tools for debugging. To access the debug logs follow these steps:
```
Comment (n-riesco) [inline TROUBLESHOOTING.md]: ~~Therefor Hydrogen provides helpful tools for debugging.~~

File: TROUBLESHOOTING.md [hunk 4]
```diff
@@ -1,3 +1,23 @@
...
+
+1. Open atom in dev mode (`View --> Developer --> Open in Dev Mode` or from the commandline via `atom --dev`):
+![dev-mode](https://cloud.githubusercontent.com/assets/13285808/22128693/30a564b6-dea2-11e6-88d3-f431a5e95aae.png)
+
+2. Open the developer tools (`View --> Developer --> Toggle Developer Tools`):
+![open-dev-tools](https://cloud.githubusercontent.com/assets/13285808/22128694/30b959d0-dea2-11e6-8b44-9c969a0b8b31.png)
+
```
Comment (n-riesco) [inline TROUBLESHOOTING.md]: How about the key bindings `shift-ctrl-i`?

**Comment by lgeiger** [comment]
@n-riesco Thanks for the feedback. I mentioned `atom --version`.

But I don't think we need the output of `apm list`, I never found it useful for our case.

**Pull request merged**